speaking-portal-project/src/api.ts

```diff
@@ -50,6 +50,9 @@ if (cluster.isPrimary) {
             // Retrieve language parameter
             // TODO: Maybe add some input validation? But its coming from kukarella so... should be fine
             const recognizer = req.body.recognizer ? req.body.recognizer : 'English (U.S.)'
+            // Retrieve character selection
+            // TODO: replace names with 'official' names for these characters
+            const characterSelect = req.body.characterSelect ? req.body.characterSelect : 'character01'
```

Comment:
Shall we also include a characterSelect into a types.ts ?

Reply:
Let's discuss this further because I think it's a great idea but we need to iron out some details! Let me know if/when you have time to go over it!

Reply:
also would like to add I really like the idea of replacing names with the official names